payloadgenerator/associated_fields.py

```diff
@@ -0,0 +1,33 @@
+from base64 import b64encode;
+from hashlib import sha1;
+from java.lang import Long;     # bug resolution import
+
+INITIAL_VALUE = 0; 
+count = INITIAL_VALUE;
+passfile_path = 'K:\\payloads.txt';
+pl_buff = list();
+lines = 0;
+for line in open(passfile_path):  
+    pl_buff.append(line.rstrip());
+    lines += 1;
+middle = str('&check=');
+NUMBER_OF_PAYLOADS = lines;
+
+def getNumberOfPayloads():
+    return Long(NUMBER_OF_PAYLOADS);    # Typecast to solve bug
```

Comment:
Right, I was asking if there was an issue in the Jython project about this, maybe http://bugs.jython.org/issue1494 ? that seems to be fixed in newer versions (following other issue reports). Did you try with the new Jython version?